src/main/java/org/joget/marketplace/TimeAgoDatalistFormatter.java

```diff
@@ -24,103 +23,118 @@ public String getVersion() {
                 } else {
-                    return Math.abs(dateDiff.getDays()) + " day(s)";
+                    return Math.abs(dateDiff.getDays()) + getDay();
                 }
 
-            } else if (duration.equals("twoDates")) {
+            } catch (Exception e) {
+               LogUtil.error("org.sample.TimeAgoDatalistFormatter", e, "Error!!");
+            }
 
-                String targetFromDate = getPropertyString("fromDate");
-                String fromDateField = (String) DataListService.evaluateColumnValueFromRow(row, targetFromDate);
-                if(fromDateField.isEmpty()){
-                    return result;
-                }
-                LocalDate fromDate = LocalDate.parse(fromDateField); //From Date
+        } else if (duration.equals("twoDates")) {
 
-                String targetToDate = getPropertyString("toDate");
-                String toDateField = (String) DataListService.evaluateColumnValueFromRow(row, targetToDate);
-                if(toDateField.isEmpty()){
-                    return result;
-                }
-                LocalDate toDate = LocalDate.parse(toDateField); //To Date
+            String targetFromDate = getPropertyString("fromDate");
+            String fromDateField = (String) DataListService.evaluateColumnValueFromRow(row, targetFromDate);
+            LocalDate fromDate = LocalDate.parse(fromDateField); // From Date
+
+            String targetToDate = getPropertyString("toDate");
+            String toDateField = (String) DataListService.evaluateColumnValueFromRow(row, targetToDate);
+            LocalDate toDate = LocalDate.parse(toDateField); // To Date
+
+            // Duration Between Two Dates
+            try {
 
-                //Duration Between Two Dates
                 dateDiff = Period.between(fromDate, toDate);
 
                 if (Math.abs(dateDiff.getYears()) > 0) {
-                    return Math.abs(dateDiff.getYears()) + " year(s) " + Math.abs(dateDiff.getMonths()) + 
-                            " month(s) " + Math.abs(dateDiff.getDays()) + " day(s)";
+                    return Math.abs(dateDiff.getYears()) + getYear() + Math.abs(dateDiff.getMonths()) +
+                            getMonth() + Math.abs(dateDiff.getDays()) + getDay();
                 } else if (Math.abs(dateDiff.getMonths()) > 0) {
-                    return Math.abs(dateDiff.getMonths()) + " month(s) " + Math.abs(dateDiff.getDays()) + " day(s)";
+                    return Math.abs(dateDiff.getMonths()) + getMonth() + Math.abs(dateDiff.getDays()) + getDay();
                 } else {
-                    return Math.abs(dateDiff.getDays()) + " day(s)";
+                    return Math.abs(dateDiff.getDays()) + getDay();
                 }
-                
+
+            } catch (Exception e) {
+                LogUtil.error("org.sample.TimeAgoDatalistFormatter", e, "Error!!");
```

Comment:
no need to change this line.

pls revert to 

LogUtil.error(TimeAgoDatalistFormatter.class.getName(), e, "Not able to compute duration");